tests/checker/good/gold/earmarked-coin.scilla.gold

```diff
@@ -5,8 +5,11 @@
     ],
     "ADT constructors": [
       {
-        "EarmarkedCoin": [
-          { "constructor": "EarmarkedCoin", "tags": [ "Money", "NotMoney" ] }
+        "earmarked-coin.EarmarkedCoin": [
+          {
+            "constructor": "earmarked-coin.EarmarkedCoin",
```

Comment:
It's not an issue the namespace `earmarked-coin` contains a hyphen, is it?

Reply:
It's not a namespace - it's an "offline" address. :-)

Since this is output from the checker for offline development it will at most make a difference to IDEs and similar tools, and they can just ban the use of hyphens in file names if they want to. 

Once the contract is being deployed the constructor will be named "<_this_address>.EarmarkedCoin", which definitely (= almost certainly) works.